### DISCUSSION TITLE
Brightness 255 vs. 254, Home Assistant, 1 step offset

### DISCUSSION THREAD

**Discussion by SaturnusDJ** [Koenkk/zigbee2mqtt#5480] [General]
Manual config in HA. All have `brightness_scale: 254`.
When setting a brightness value, the value reported back is one lower starting from (when setting) brightness 128.
E.g.
```
127 --> 127
128 --> 127
254 --> 253
255 --> 254 (max)
```
Changing `brightness_scale:` to `255` messed things up in the other half (0-128).

In https://github.com/Koenkk/zigbee2mqtt/issues/4163 I saw this topic was touched, but I miss a definitive conclusion. Is this Z2M sticking to the industry standard and HA not doing that? Thereby, is it impossible to prevent this annoying difference?

**Comment by Koenkk** [comment]
The max brightness for Zigbee is 254, therefore Zigbee2MQTT sticks to that.

**Reply by SaturnusDJ** [reply]
Thanks. Thought so. Will check with Home Assistant if their value range could be made adjustable.
